src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/PushCommand.cs

```diff
@@ -98,7 +98,7 @@ public static void Register(CommandLineApplication app, Func<ILogger> getLogger)
                         throw new ArgumentException(Strings.Push_InvalidTimeout);
                     }
 
-                    var sourceProvider = new PackageSourceProvider(XPlatUtility.CreateDefaultSettings());
+                    var sourceProvider = new PackageSourceProvider(XPlatUtility.CreateDefaultSettings(), enablePackageSourcesChangedEvent: true);
```

Comment:
This one seems weird. 
Do these events fire in commandline scenarios?

Reply:
no, it was a mistake.